packages/web/src/components/edit/fields/RemixSettingsField/TrackInfo.tsx

```diff
@@ -34,7 +34,9 @@ export const TrackInfo = (props: TrackInfoProps) => {
       />
       <Text variant='body' strength='strong'>
         {track.title} <Text color='subdued'>{messages.by}</Text>{' '}
-        <UserLink userId={track.owner_id} />
+        <Box css={{ display: 'inline-flex' }}>
```

Comment:
@dharit-tan @Kyle-Shanks lets have a debrief on the user-link changes with width 100%. it's led to quite a few regressions